qngng/qngng.py

```diff
@@ -73,6 +78,9 @@ def surname(self):
     def gender(self):
         return self._gender
 
+    @property
+    def middle_name_initial(self):
+        return self._middle_name[0]
```

Comment:
Maybe append `.upper()` just in case.

Reply:
Just `middle_initial()`.

Also, add:

    assert self._middle_name is not None